src/registrar/models/domain_application.py

```diff
@@ -29,14 +29,14 @@ class DomainApplication(TimeStampedModel):
     REJECTED = "rejected"
     INELIGIBLE = "ineligible"
     STATUS_CHOICES = [
-        (STARTED, STARTED),
-        (SUBMITTED, SUBMITTED),
-        (IN_REVIEW, IN_REVIEW),
-        (ACTION_NEEDED, ACTION_NEEDED),
-        (APPROVED, APPROVED),
-        (WITHDRAWN, WITHDRAWN),
-        (REJECTED, REJECTED),
-        (INELIGIBLE, INELIGIBLE),
+        (STARTED, STARTED.capitalize()),
```

Comment:
Same as my comment on the JS; I think a structural revision here to use tuples (as you did for Domain states) can propagate through the app and would be more solid:

```
    # Constants for choice fields
    class ApplicationStatus(models.TextChoices):
        STARTED = "started", "Started"
        SUBMITTED = "submitted", "Submitted"
        IN_REVIEW = "in_review", "In review"
        ACTION_NEEDED = "action_needed", "Action needed"
        APPROVED = "approved", "Approved"
        WITHDRAWN = "withdrawn", "Withdrawn"
        REJECTED = "rejected", "Rejected"
        INELIGIBLE = "ineligible", "Ineligible"
```

This would require revising the references to DA statuses throughout the app AND an edit to admin.py to display the verbose name.

As I was doing the review I played around the codebase a bit. I'll make a branch.